scss/main.scss

```diff
@@ -1 +1,13 @@
 @use 'vars';
+
```

Comment:
lets add all the variables from vars.scss to here for now and use those :-----')